plugin/helpers.go

```diff
@@ -18,6 +18,36 @@ import (
 // DefaultTimeout is the default timeout for prefetching
 var DefaultTimeout = 10 * time.Second
 
+/*
+CName returns the entry's canonical name, which is what Wash uses
+to construct the entry's path (see plugin.Path). The entry's cname
+is e.Name(), but with all '/' characters replaced by a '#' character.
```

Comment:
I decided to go ahead and do the `#` replacement because it simplifies the CName semantics.